apps/researcher/src/app/[locale]/objects/[id]/page.tsx

```diff
@@ -15,6 +24,53 @@ interface Props {
   params: {id: string};
 }
 
+function ContactDetailsSlideOver({datasetId}: {datasetId?: string}) {
+  const {organization} = useCurrentPublisher.getState();
+  const t = useTranslations('ObjectDetails');
+
+  if (!organization) {
+    return (
+      <SlideOverDialog>
+        <SlideOverHeader />
+        <SlideOverContent>
+          <div className="flex-col gap-4 mt-4 flex">
+            <h2>{t('noOrganizationFound')}</h2>
+          </div>
+        </SlideOverContent>
+      </SlideOverDialog>
+    );
+  }
+
+  return (
+    <SlideOverDialog>
+      <SlideOverHeader />
+      <SlideOverContent>
+        <div className="flex-col gap-4 mt-4 flex">
+          <h2>{organization.name}</h2>
+          <span>
+            {organization.address?.streetAddress}
+            <br /> {organization.address?.postalCode}{' '}
+            {organization.address?.addressLocality},
+            <br /> {organization.address?.addressCountry}
+          </span>
+          {organization.url && (
+            <a href={organization.url}>{t('linkToWebsite')}</a>
+          )}{' '}
+          {datasetId && (
+            <a
+              href={`${
+                process.env['DATASET_BROWSER_URL']
```

Comment:
Would it be possible to first `import {env} from 'node:process'` and then to use `env['DATASET_BROWSER_URL']` (see e.g. https://2ality.com/2021/12/node-protocol-imports.html)?